[
  {
    "url": "https://api.github.com/repos/electron/electron/issues/comments/182573485",
    "html_url": "https://github.com/electron/electron/issues/4396#issuecomment-182573485",
    "issue_url": "https://api.github.com/repos/electron/electron/issues/4396",
    "id": 182573485,
    "user": {
      "login": "deepak1556",
      "id": 964386,
      "avatar_url": "https://avatars2.githubusercontent.com/u/964386?v=3",
      "gravatar_id": "",
      "url": "https://api.github.com/users/deepak1556",
      "html_url": "https://github.com/deepak1556",
      "followers_url": "https://api.github.com/users/deepak1556/followers",
      "following_url": "https://api.github.com/users/deepak1556/following{/other_user}",
      "gists_url": "https://api.github.com/users/deepak1556/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/deepak1556/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/deepak1556/subscriptions",
      "organizations_url": "https://api.github.com/users/deepak1556/orgs",
      "repos_url": "https://api.github.com/users/deepak1556/repos",
      "events_url": "https://api.github.com/users/deepak1556/events{/privacy}",
      "received_events_url": "https://api.github.com/users/deepak1556/received_events",
      "type": "User",
      "site_admin": false
    },
    "created_at": "2016-02-10T20:37:12Z",
    "updated_at": "2016-02-10T20:37:12Z",
    "body": "Can confirm the behavior for 303 responses. Related to https://github.com/atom/electron/issues/3471 .\n"
  },
  {
    "url": "https://api.github.com/repos/electron/electron/issues/comments/186920417",
    "html_url": "https://github.com/electron/electron/issues/4396#issuecomment-186920417",
    "issue_url": "https://api.github.com/repos/electron/electron/issues/4396",
    "id": 186920417,
    "user": {
      "login": "aggregatejeff",
      "id": 6768600,
      "avatar_url": "https://avatars3.githubusercontent.com/u/6768600?v=3",
      "gravatar_id": "",
      "url": "https://api.github.com/users/aggregatejeff",
      "html_url": "https://github.com/aggregatejeff",
      "followers_url": "https://api.github.com/users/aggregatejeff/followers",
      "following_url": "https://api.github.com/users/aggregatejeff/following{/other_user}",
      "gists_url": "https://api.github.com/users/aggregatejeff/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/aggregatejeff/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/aggregatejeff/subscriptions",
      "organizations_url": "https://api.github.com/users/aggregatejeff/orgs",
      "repos_url": "https://api.github.com/users/aggregatejeff/repos",
      "events_url": "https://api.github.com/users/aggregatejeff/events{/privacy}",
      "received_events_url": "https://api.github.com/users/aggregatejeff/received_events",
      "type": "User",
      "site_admin": false
    },
    "created_at": "2016-02-21T21:20:06Z",
    "updated_at": "2016-02-21T21:22:31Z",
    "body": "Any update on this? I can certainly work around it by just ignoring errorcode -3, but that's clearly not ideal. Also, is there a list of returned errorcodes anywhere?\n"
  },
  {
    "url": "https://api.github.com/repos/electron/electron/issues/comments/205640320",
    "html_url": "https://github.com/electron/electron/issues/4396#issuecomment-205640320",
    "issue_url": "https://api.github.com/repos/electron/electron/issues/4396",
    "id": 205640320,
    "user": {
      "login": "bmcminn",
      "id": 879984,
      "avatar_url": "https://avatars0.githubusercontent.com/u/879984?v=3",
      "gravatar_id": "",
      "url": "https://api.github.com/users/bmcminn",
      "html_url": "https://github.com/bmcminn",
      "followers_url": "https://api.github.com/users/bmcminn/followers",
      "following_url": "https://api.github.com/users/bmcminn/following{/other_user}",
      "gists_url": "https://api.github.com/users/bmcminn/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/bmcminn/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/bmcminn/subscriptions",
      "organizations_url": "https://api.github.com/users/bmcminn/orgs",
      "repos_url": "https://api.github.com/users/bmcminn/repos",
      "events_url": "https://api.github.com/users/bmcminn/events{/privacy}",
      "received_events_url": "https://api.github.com/users/bmcminn/received_events",
      "type": "User",
      "site_admin": false
    },
    "created_at": "2016-04-05T04:35:56Z",
    "updated_at": "2016-04-05T04:35:56Z",
    "body": "@aggregatejeff, you mean something like this? https://src.chromium.org/svn/trunk/src/net/base/net_error_list.h\n"
  },
  {
    "url": "https://api.github.com/repos/electron/electron/issues/comments/217303129",
    "html_url": "https://github.com/electron/electron/issues/4396#issuecomment-217303129",
    "issue_url": "https://api.github.com/repos/electron/electron/issues/4396",
    "id": 217303129,
    "user": {
      "login": "CharlieHess",
      "id": 1865957,
      "avatar_url": "https://avatars2.githubusercontent.com/u/1865957?v=3",
      "gravatar_id": "",
      "url": "https://api.github.com/users/CharlieHess",
      "html_url": "https://github.com/CharlieHess",
      "followers_url": "https://api.github.com/users/CharlieHess/followers",
      "following_url": "https://api.github.com/users/CharlieHess/following{/other_user}",
      "gists_url": "https://api.github.com/users/CharlieHess/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/CharlieHess/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/CharlieHess/subscriptions",
      "organizations_url": "https://api.github.com/users/CharlieHess/orgs",
      "repos_url": "https://api.github.com/users/CharlieHess/repos",
      "events_url": "https://api.github.com/users/CharlieHess/events{/privacy}",
      "received_events_url": "https://api.github.com/users/CharlieHess/received_events",
      "type": "User",
      "site_admin": false
    },
    "created_at": "2016-05-05T22:53:06Z",
    "updated_at": "2016-05-05T22:53:06Z",
    "body": "We see the same thing during the Slack sign-in flow. We explicitly ignore that error code, but it feels bad. I've clarified the documentation [here](https://github.com/electron/electron/pull/5409).\n"
  },
  {
    "url": "https://api.github.com/repos/electron/electron/issues/comments/220857481",
    "html_url": "https://github.com/electron/electron/issues/4396#issuecomment-220857481",
    "issue_url": "https://api.github.com/repos/electron/electron/issues/4396",
    "id": 220857481,
    "user": {
      "login": "tbo",
      "id": 264745,
      "avatar_url": "https://avatars1.githubusercontent.com/u/264745?v=3",
      "gravatar_id": "",
      "url": "https://api.github.com/users/tbo",
      "html_url": "https://github.com/tbo",
      "followers_url": "https://api.github.com/users/tbo/followers",
      "following_url": "https://api.github.com/users/tbo/following{/other_user}",
      "gists_url": "https://api.github.com/users/tbo/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/tbo/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/tbo/subscriptions",
      "organizations_url": "https://api.github.com/users/tbo/orgs",
      "repos_url": "https://api.github.com/users/tbo/repos",
      "events_url": "https://api.github.com/users/tbo/events{/privacy}",
      "received_events_url": "https://api.github.com/users/tbo/received_events",
      "type": "User",
      "site_admin": false
    },
    "created_at": "2016-05-22T21:27:22Z",
    "updated_at": "2016-05-22T21:46:23Z",
    "body": "@CharlieHess How exactly do you ignore that error code? Once the error occurs I'm unable to load another URL in the same webview.\n"
  },
  {
    "url": "https://api.github.com/repos/electron/electron/issues/comments/226950986",
    "html_url": "https://github.com/electron/electron/issues/4396#issuecomment-226950986",
    "issue_url": "https://api.github.com/repos/electron/electron/issues/4396",
    "id": 226950986,
    "user": {
      "login": "ccnokes",
      "id": 1610343,
      "avatar_url": "https://avatars3.githubusercontent.com/u/1610343?v=3",
      "gravatar_id": "",
      "url": "https://api.github.com/users/ccnokes",
      "html_url": "https://github.com/ccnokes",
      "followers_url": "https://api.github.com/users/ccnokes/followers",
      "following_url": "https://api.github.com/users/ccnokes/following{/other_user}",
      "gists_url": "https://api.github.com/users/ccnokes/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/ccnokes/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/ccnokes/subscriptions",
      "organizations_url": "https://api.github.com/users/ccnokes/orgs",
      "repos_url": "https://api.github.com/users/ccnokes/repos",
      "events_url": "https://api.github.com/users/ccnokes/events{/privacy}",
      "received_events_url": "https://api.github.com/users/ccnokes/received_events",
      "type": "User",
      "site_admin": false
    },
    "created_at": "2016-06-18T16:20:59Z",
    "updated_at": "2016-06-18T16:20:59Z",
    "body": "@aggregatejeff @bmcminn I got tired of constantly looking at that chrome header file for what the error is, so I made: https://github.com/ccnokes/chrome-network-errors.\n"
  }
]